src/utils/size-utils.ts

```diff
@@ -0,0 +1,37 @@
+import { Theme } from '../theme';
+
+type Size = 'md' | 'sm';
```

Comment:
this as we are using to much can be defined to a theme/sizing file

Reply:
ok, I will pass a description comment